README.md

```diff
@@ -23,6 +25,11 @@
 * LinkdIn: [@aneallaryea](https://www.linkedin.com/in/nii-aneal-84ba7a147)
 * Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 
+👤 Andy Menutti
+* Github: [@andym80](https://github.com/andym80)
+* LinkdIn: [@andym80](https://shorturl.at/cADGM)
+
+
 
```

Comment:
- [ **OPTIONAL** ] Good job :+1: :100:  making all changes required by the previous reviewer. But, I think removing unnecessary code lines would be better to make your documentation code look cleaner.